charts/seed-monitoring/charts/grafana/templates/_helpers.tpl

```diff
@@ -82,7 +82,7 @@ grafana-{{ .Values.role }}-datasources-{{ include "grafana.datasources.data" . |
 {{ end }}
 {{ end }}
 {{- if .Values.extensions.dashboards }}
-{{- toString .Values.extensions.dashboards | indent 2 }}
+{{- toString .Values.extensions.dashboards }}
```

Comment:
I'm not sure, if I understand how a changed indentation fixes "missing" values.
Can you elaborate on what the bug exactly was?

Reply:
The data items for the extensions dashboards have been added to the yaml with indentation of 4 spaces.

For example:
```
apiVersion: v1
data:
  apiserver-admission-details.json: |-
…
  vpn-dashboard.json: |-
    {
…
    }
 
    calico-felix-dashboard.json: |-
      {
…
      }
```

Here `apiserver-admission-details.json` and `vpn-dashboard.json` are predefined dashboards and correctly indented. But the extension dashboard `calico-felix-dashboard.json` is indented with 4 spaces.
The predefined dashboards are added with two parts. The `{{ base $name }}` is added without indent and only the value part is indented. The error is that the extension dashboard is added as one string consisting of both key and value and therefore needs no indentation.

Reply:
Ahh, thanks for explaining. That makes sense :)